package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Sun Nov  8 20:55:53 UTC 2015 - mfilka@suse.com
+
+- bnc#944349
+  - do not set links down when installing over network
```

Comment:
The code looks good, but we should improve the explanation:

- "when installing over network" is too broad. HTTP? FTP? VNC?
- This describes *what* is done. But as we are changing a notoriously complex thing, we should say *why*.